gax-java/gax/src/main/resources/META-INF/native-image/com.google.api/gax/native-image.properties

```diff
@@ -3,7 +3,8 @@ Args = --enable-url-protocols=https,http \
   org.junit.platform.engine.TestTag,\
   com.google.api.gax.core.GaxProperties,\
   com.google.common.base.Platform,\
-  com.google.common.base.Platform$JdkPatternCompiler \
+  com.google.common.base.Platform$JdkPatternCompiler,\
+  com.google.protobuf.RuntimeVersion \
```

Comment:
Since this is a class only exist in protobuf 4.x, if a customer overrides protobuf to 3.x, I'm assuming it is going to fail for them?
If that's the case, we need to have a different solution, because our SDK is now guaranteed to be compatible with both protobuf 3.x and 4.x.

Reply:
Answered offline that it is not an issue because the classes are not going to be loaded if they don't exist.